client/modules/users/authReducer.js

```diff
@@ -30,6 +36,32 @@ export const clearUser = () => ({type: CLEAR_USER})
 
 export const clearFlags = () => ({type: CLEAR_FLAGS})
 
+export const loadNotifications = () => ({
+  [CALL_API]: {
+    endpoint: '/users/me/listnotifications',
+    schema: arrayOfNotifications,
+    types: [actions.LOAD_ALL_REQUEST, actions.LOAD_ALL_SUCCESS, actions.LOAD_ALL_FAILURE]
```

Comment:
These are actions for load/delete a user and shouldn't be reused for this in case something else is listening for them, just add some new ones to the top of the file